src/components/Report.tsx

```diff
@@ -33,7 +33,6 @@ type AuditInfo = Partial<{
   type: string;
   fileName: string;
   filePath: string;
-  impactedWorkflows: string[];
```

Comment:
why there was a removal of impactedWorkflows? Need more context

Reply:
impacted workflow was removed in [recent PR](PR), this seems to have just been forgotten to be removed